bin/whoami

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/perl
+
+=encoding utf8
+
+=begin metadata
+
+Name: whoami
+Description: display effective user ID
+Author: Oğuz Ersen, oguzersen@protonmail.com
+License: artistic2
+
+=end metadata
+
+=cut
+
+use strict;
+
+my $user;
+
+if ($^O eq 'MSWin32') {
+    require Win32;
+    $user = Win32::LoginName();
```

Comment:
handle the case where Win32::* is not available. See the `getlogin()` that I showed you earlier. Remember, you are mostly limited to core modules.

Reply:
I changed this part with `$user = getlogin() || $ENV{USERNAME};`, Is this enough or are there edge cases where both `getlogin()` returns empty and `USERNAME` environment variable is not available? I can test it as soon as I have access to a Windows machine but it may take a while.

Reply:
Once you get a chance to work with this on Windows, see what works.

But, if you don't have access to Windows, is there some other reason you'd like this program to be in PerlPowerTools? Is there another non-unix system you are targeting that doesn't already have a `whoami`?

Reply:
I just saw it was missing and wanted to contribute (: This can be closed without merging if it is not suitable for PerlPowerTools. Thanks for your guidance and patience.